ProcessLib/ComponentTransport/ComponentTransportFEM.h

```diff
@@ -359,15 +467,28 @@ class LocalAssemblerData : public ComponentTransportLocalAssemblerInterface
             return shape_matrices;
         }();
 
+        // concentration index of the component ranking first
+        auto const concentration_index = 1 * ShapeFunction::NPOINTS;
+        // assuming that fluid density always depends on concentration of the
+        // first component.
+        // get local_C and local_p
```

Comment:
Drop this line. The comment is useless above (where it is correct). Here it is wrong. Don't copy-paste comments.

Reply:
Removed.